docs/cli/overview.md

```diff
@@ -1,23 +1,40 @@
 # Overview
 
-Walking side by side with the Python client, the Inductiva CLI provides 
-a quick and simple interface to interact and perform the most common tasks from
-the terminal. The CLI allows users to quickly manage their tasks, resources and/or
-storage, while their simulations are still running remotely.
+The Inductiva Command Line Interface (CLI) offers a streamlined and efficient way 
+to interact with our platform directly from your terminal. Designed for ease of use, 
+the CLI enables you to manage simulations, resources, and data in real-time, even 
```

Comment:
comma before final enumeration:

```suggestion
the CLI enables you to manage simulations, resources and data in real-time, even 
```

Also, not sure the final adverb is applicable 🤔

Reply:
@sergiomsantos you mean real-time?